docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -2,146 +2,173 @@
-  2. When using an API Key, no username needs to be provided. This is because the API Key maps to a ServiceId, and that ServiceId is mapped to the Db2 user.
+    - Note (1): The page forces the value to lower-case, so `ServiceId` becomes `serviceid`. Don't be alarmed, it still works.
 
-### Configuring a Liberty Datasource with API Key
+    - Note (2): Do NOT select Administrator. One should follow the least-privelege principal for the FHIRSERVER user.
 
-The IBM FHIR Server Import/Export feature utilizes Java Batch (JSR-352) provided by the batch-1.0 feature in Liberty Profile. The persistence layer can be configured to use Db2 as follows:
+1. Click `Create`.
 
-Create a Db2 user (e.g. FHIRBATCH) and associate it with a ServiceId (no need to create an Administration user, a simple user has sufficient privileges). Using a valid API-KEY for the given ServiceId, configure a new datasource and the Java Batch persistence layer as follows:
+You are now able to connect to the database as the FHIRSERVER user using only the API key created above.
 
-``` xml
-    <dataSource id="fhirbatchDS" jndiName="jdbc/fhirbatchDB">
-        <jdbcDriver libraryRef="fhirSharedLib"/>
-        <properties.db2.jcc
-            serverName="dashdb-txn-flex-************.services.dal.bluemix.net"
-            portNumber="50001"
-            apiKey="<API-KEY>"
-            securityMechanism="15"
-            pluginName="IBMIAMauth"
-            databaseName="BLUDB"
-            currentSchema="JBATCH"
-            driverType="4" sslConnection="true" sslTrustStoreLocation="resources/security/dbTruststore.jks" sslTrustStorePassword="<TRUSTSTORE-PASSWORD>"/>
-    </dataSource>
+### **Testing the Connection**
 
-    <batchPersistence jobStoreRef="BatchDatabaseStore" />
-    <databaseStore id="BatchDatabaseStore" dataSourceRef="fhirbatchDS" schema="JBATCH" tablePrefix="" />
-```
+The [Db2 driver](https://repo1.maven.org/maven2/com/ibm/db2/jcc/11.5.0.0/jcc-11.5.0.0.jar) is able to execute a connectivity test to check the configuration of the combo of API-key/Service-Id/Db2-User-Id.
+
+1. Copy the command to your code editor
 
-Note, the Java Batch is configured in batchDs.xml  and included from the default server.xml that gets installed to the `{wlp}/usr/server/fhir-server`.
+    ``` bash
+    java -cp /path/to/db2jcc4.jar com.ibm.db2.jcc.DB2Jcc  -url "jdbc:db2://<DB2-HOSTNAME>:50001/BLUDB:apiKey=<API-KEY>;securityMechanism=15;sslConnection=true;sslTrustStoreLocation=/path/to/truststore.jks;sslTrustStorePassword=<TRUSTSTORE-PASSWORD>;"
+    ```
 
-### Configuring FHIR Datasource
+    - Note: Don't forget the trailing `;` in the URL. Some of the documented examples don't include it, but it is required in order for the connection to work, although this may be fixed in a future driver release. This only affects this test URL, not the actual FHIR server configuration.
```

Comment:
wow, interesting

Reply:
yeah - I left it there (originally from Robin)